### PR TITLE
Import config

### PR DESCRIPTION
**Pull request by mt-yucheng**
- Imported Rubocop configs from one of our major projects.
- Locked Rubocop version at `0.70.0`

## WARNING

This repository is public. Be sure to conceal company secrets when posting discussions in this pull request.

## Pre-Requisites

- [x] Confirme License is acceptable

## Post-Requisites

- [ ] Tag `v1.0.0`

### CODE REVIEW

File: rubocop.rails.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  Exclude:
+    - 'bin/**/*'
+    - 'db/schema.rb'
+
+Rails/Output:
+  Enabled: true
+
+Metrics/BlockLength:
+  Exclude:
+    # ActiveAdmin's DSL prefers long blocks
```
Comment (makdad) [inline rubocop.rails.yml:15]: indeed it does....

File: rubocop.rails.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    - app/admin/**/*
+    - config/**/*
+
+Style/Documentation:
+  Exclude:
+    # Controller classes are considered self-documented.
```
Comment (makdad) [inline rubocop.rails.yml:21]: アグリー

File: README.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
+# Moneytree Rubocop Config
+
+Shared Rubocop config used by Moneytree's Ruby / Rails projects.
```
Comment (makdad) [inline README.md:3]: If this is going to be open source, maybe a little more text here might be nice -- it's also an advertisement for new devs.  We might want to talk a little about how we value clean code, and we publish what we like in Ruby.

**Review by makdad — APPROVED** [review]
From a tech perspective, LGTM.

File: README.md (hunk 3 above)
Comment (mt-yucheng) [reply README.md:3]: I've added some words below this line. Could you take a look? (Especially for grammar, please)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+### Why
+
+At Moneytree, many of our services are build on Ruby and Rails. As our team
```
Comment (mt-yucheng) [inline README.md]:
```suggestion
At Moneytree, many of our services are built on Ruby and Rails. As our team
```

File: README.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+grows larger, it is necessary to keep all our codes in similar coding styles,
+while providing some level of flexibility to developers.
+
+Instead of sticking with a particular coding style, and potentially introduce
+endless discussion about which one is better, we value:
+
+1. Less Error-Prone - Is one style easier to cause runtime error than another?
+2. Code Readability - Is one style is more readable than another?
```
Comment (mt-yucheng) [inline README.md]:
```suggestion
2. Code Readability - Is one style more readable than another?
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+3. Easy On-Boarding - Is it easy for new developers to get used to this coding style?
+4. Shared Knowledge - Does this coding style apply to most of our active projects?
+
+### We're Hiring
+
+Interested in working at the cutting-edge FinTech industry in Japan? Please
```
Comment (makdad) [inline README.md:21]: 👍 👍 👍 👍

**Review by makdad — APPROVED** [review]
looks good (README changes)

File: README.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+    - Exclude
+```
+
+## Development
+
+TODO: write development instructions here
+
+## Contributing
+
+Bug reports and pull requests are welcome on GitHub at https://github.com/[USERNAME]/moneytree-rubocop-config.
```
Comment (mt-ronkorving) [inline README.md]: I guess this needs an update?

**Comment by mt-ronkorving** [comment]
@mt-yucheng Our CTO @rsharrott has signed off on using MIT, so you can tick that box.

File: README.md (hunk 7 above)
Comment (mt-yucheng) [reply README.md]:
```suggestion
Bug reports and pull requests are welcome on GitHub at https://github.com/moneytree/moneytree-rubocop-config.
```

Comment (mt-yucheng) [reply README.md]: Thanks for reminder!

File: test/app/admin/user.rb [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+  long
+  long
+  long
+  long
+  long
+  long
+  long
+  long
+  long
+  long
```
Comment (mt-james) [inline test/app/admin/user.rb:45]: image

**Review by mt-james — APPROVED** [review]
I like it. Everything seems about right to me

File: rubocop.yml [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+  Max: 120
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented_relative_to_receiver
+
+Layout/IndentFirstHashElement:
+  EnforcedStyle: consistent
+
+Metrics/AbcSize:
+  Max: 15
```
Comment (mt-tomohiko) [inline rubocop.yml:17]: This could possibly be too small for a major part of our codebase, though.

**Review by mt-tomohiko — APPROVED** [review]

File: rubocop.yml (hunk 9 above)
Comment (mt-yucheng) [reply rubocop.yml:17]: This is from the default value of Rubocop. I want to keep it for new projects, but of course we can always override this value for legacy projects :)

**Pull request merged**